docs/reference/reference-training/training/api-pytorch-trainer-reference.rst

```diff
@@ -0,0 +1,120 @@
+###############################
+ PyTorch Trainer API Reference
+###############################
+
++----------------------------------------------------+
+| User Guide                                         |
++====================================================+
+| :doc:`/training/apis-howto/api-pytorch-trainer-ug` |
++----------------------------------------------------+
+
+.. code::
+
+   determined.pytorch.init(
+       *,
+       hparams: Optional[Dict] = None,
+       exp_conf: Optional[Dict[str, Any]] = None,
+       distributed: Optional[core.DistributedContext] = None,
+       aggregation_frequency: int = 1,
+   ) -> pytorch.PyTorchTrialContext:
+
+``pytorch.init()`` builds a ``pytorch.PyTorchTrialContext`` for use with ``PyTorchTrial``.
+
+Always use this method to construct a ``PyTorchTrialContext`` instead of instantiating the class
+directly.
+
+All of the arguments are optional, but ``hparams`` and ``exp_conf`` are used to set the
+corresponding variables on ``PyTorchTrialContext``. So if not passed in, calling
+``context.get_hparams()`` or ``context.get_experiment_config()`` will fail in local-training mode.
```

Comment:
> will fail in local-training mode

@azhou-determined wait I thought we settled on no magics?  Wouldn't these calls always break if hparams/exp_conf aren't set?